docs/scos/dev/glue-api-guides/202311.0/conventions/default-conventions.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Default conventions
+description: This document will describe what convention Spryker provides OOTB
+last_updated: December 12, 2023
+template: howto-guide-template
+---
+
+Spryker provides some of the [conventions](/docs/scos/dev/glue-api-guides/{{page.version}}/conventions/what-is-api-convention.html) OOTB
+
+## Available conventions
+
+### JSON:API
+
+JSON:API is available for usage OOTB in Spryker. In order to use it you need to implement the `Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\JsonApiResourceInterface` interface in your resource plugin.
+If you need to extend it behaviour or add missing functionality you can add additional plugins in `\Pyz\Glue\GlueJsonApiConvention\GlueJsonApiConventionDependencyProvider`. All provided stacks are using the interfaces from `GlueApplicationExtension` modules and works in the same way as the similar plugins in specific application modules.
```

Comment:
What does this mean "works in the same way as the similar plugins in specific application modules.
Comment". This information is usless as long to do not understand GlueApplicationExtension please add references or explain.